src/quo2/components/common/not_implemented.cljs

```diff
@@ -0,0 +1,12 @@
+(ns quo2.components.common.not-implemented
+  (:require [react-native.core :as rn]))
+
+(defn not-implemented
+  [content]
+  (when content
+    [rn/view {:border-color :red :border-width 1}
```

Comment:
These styles are not inside a `:style` prop.

Reply:
Done